firmware/Core/Src/antenna_deploy_drivers/ant_commands.c

```diff
@@ -181,6 +181,13 @@ uint8_t ANT_CMD_measure_temp(uint16_t *result) {
     }
     return comms_err;
 }
+/// @brief  Converts temperature measurement to centi-degrees celsius
+/// @param measurement measurement result from ANT_CMD_measure_temp()
+/// @return returns the temperature in centi-degrees celsius
+/// @note The formula used is based on the formula found in the data sheet for the LM94022. Derivation can be found on github issue#146.
+int16_t ANT_CMD_convert_temp_measurement_to_centi_degree_celsius(uint16_t measurement) {
```

Comment:
This function isn't a command, so should be prefixed without `CMD_`. 

Change name to `ANT_convert_temp_measurement_to_centi_degree_celsius`.